lib/app/utils/fhir_models_validator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Inferno
+  # FHIRModelsValidator extends BaseValidator to use the validation in fhir_models.
+  # It passes the validation off to the correct model version.
+  class FHIRModelsValidator
+    def initialize; end
```

Comment:
Is this actually needed?

Reply:
No, it's not; it was there when the validator classes used to inherit from a parent class, but now that they don't this can be removed.